package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "ai-pills",
+  "version": "1.0.0",
+  "description": "💊 AI Pills - Quick and clear tips related to Artificial Intelligence",
+  "main": "index.js",
+  "scripts": {
```

Comment:
I will suggest to make the package private, `private: true` to avoid accidental publication.